ui/page/dexclient/password_modal.go

```diff
@@ -0,0 +1,114 @@
+package dexclient
+
+import (
+	"gioui.org/layout"
+	"gioui.org/widget"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+const dexPasswordModalID = "dex_password_modal"
+
+type passwordModal struct {
+	*load.Load
+	modal *decredmaterial.Modal
+
+	createPassword                decredmaterial.Button
+	appPassword, appPasswordAgain decredmaterial.Editor
+
+	isSending    bool
+	appInitiated func()
+}
+
+func newPasswordModal(l *load.Load) *passwordModal {
+	md := &passwordModal{
+		Load:             l,
+		modal:            l.Theme.ModalFloatTitle(),
+		appPassword:      l.Theme.EditorPassword(new(widget.Editor), "Password"),
+		appPasswordAgain: l.Theme.EditorPassword(new(widget.Editor), "Password again"),
+		createPassword:   l.Theme.Button("Create password"),
+	}
+
+	md.createPassword.TextSize = values.TextSize12
+	md.createPassword.Background = l.Theme.Color.Primary
+	md.appPassword.Editor.SingleLine = true
+	md.appPasswordAgain.Editor.SingleLine = true
+
+	return md
+}
+
+func (md *passwordModal) ModalID() string {
+	return dexPasswordModalID
+}
```

Comment:
Why do we have a new password modal?.. We currently have password modals in the modal package.

Reply:
This modal have two inputs for password and re-password

Reply:
Should we change it to a page?